x-pack/plugins/security_solution/public/common/components/navigation/use_security_solution_navigation/index.test.tsx

```diff
@@ -117,6 +122,23 @@ describe('useSecuritySolutionNavigation', () => {
     ).toBeUndefined();
   });
 
+  it('should omit response actions history if hook reports false', () => {
```

Comment:
being that the `useUserPrivileges` is being mocked via `jest.mock()`, should you add a `afterEach()` that resets the mock back to its default?

Reply:
oh yes.  Missed that. Will add it.

Reply:
3d93a26bdabb34feedceaaba79f354c48d1ac75f